src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -952,6 +952,19 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         goto error;
     }
 
+    /* Allocate memory for source and sink names. */
+    stream->sourceStreamName = (char*)PaUtil_AllocateMemory(strlen("Portaudio source"));
+    stream->sinkStreamName = (char*)PaUtil_AllocateMemory(strlen("Portaudio sink"));
```

Comment:
The string should become a const char[] to avoid duplication. 
Than you can use sizeof().

Reply:
Updated to use sizeof().